subsys/net/l2/ieee802154/ieee802154_mgmt.c

```diff
@@ -430,6 +428,9 @@ static int ieee802154_associate(uint32_t mgmt_request, struct net_if *iface,
 	}
 
 out:
+	if (ret < 0)
+		ieee802154_filter_pan_id(iface, 0);
```

Comment:
Zephyr requires surrounding `{ }` even for single-line blocks (weird that checkpatch did not catch this).
```suggestion
	if (ret < 0) {
		ieee802154_filter_pan_id(iface, 0);
	}
```

Reply:
Oh, right, I'm sorry I'm used to the Linux kernel coding style where this is not necessary, I've been warned by checkpatch somewhere else about this but I missed this one.